### DISCUSSION TITLE
Communication between view model

### DISCUSSION THREAD

**Discussion by ebabhi** [FilledStacks/stacked#975] [Q&A]
can 2 view model communicate directly? if yes can you please provide me an example

**Comment by zuboje** [comment]
> can 2 view model communicate directly? if yes can you please provide me an example

You need to provide an example for what you are trying to achieve, but in general you should use reactive values that are shared between view models and change values as needed. There are examples of reactive view models and reactive values https://stacked.filledstacks.com/docs/basics-in-depth/viewmodels/#reactiveviewmodel

**Comment by ferrarafer** [comment]
In general, when you get into that situation is because you need to create a Service to handle some functionality. As @zuboje said, please provide your use case so we can know what you want to do and we can try to find the best implementation.
